contracts/utils/Timed.sol

```diff
@@ -63,6 +63,12 @@ abstract contract Timed {
         emit TimerReset(block.timestamp);
     }
 
+    function _pauseTimer() internal {
```

Comment:
I don't like adding on to existing contracts that are already deployed. I'd prefer extending the timed class to include this helper method over modifying it.

Reply:
Agreed with @ElliotFriedman here

Reply:
changed